1-the-basics/style.css

```diff
@@ -0,0 +1,23 @@
+blockquote {
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+
+button:hover {
+    background-color: palevioletred;
+}
+
+footer, header {
+    background-color: peachpuff;
+}
+
+form {
+    background-image: url(https://1.bp.blogspot.com/-aOdYJd_BI9E/UfGjBm5OmdI/AAAAAAAAErc/KFPcoQZ3q00/s1600/wallpapers_tecnologia_computacion_by_madboxpc-1.png);
+    background-size: cover;
+    background-color: salmon;
```

Comment:
- Set a background-image to the **element that contain**s the `form`.
- Change the background-color of the <form>.

In that way, you will see only one of the rules.